src/AutorestSwift/View Models/ExceptionResponseViewModel.swift

```diff
@@ -32,26 +32,26 @@ struct ExceptionResponseViewModel {
     let objectType: String?
     /// Identifies the correct snippet to use when rendering the view model
     let description: String?
+    let isDefaultStatusCode: Bool
 
     init(from response: Response) {
         let httpResponse = response.protocol.http as? HttpResponse
         var statusCodes = [String]()
         httpResponse?.statusCodes.forEach { statusCodes.append($0.rawValue) }
 
-        self.statusCodes = statusCodes
+        self.statusCodes = statusCodes.filter { $0 != "default" }
```

Comment:
What happens if it is default? This ends up being an empty collection? Is that the intention? What defines what the defaults are?

Reply:
I just saw your explanation. It does seem then that this collection would always just be integers.

Reply:
When it is default and there is a body, the handling will be generated in OperationDefaultExceptionSnippet.stencil